tests/cli/commands/test_db_command.py

```diff
@@ -37,8 +37,11 @@ def setup_class(cls):
 
     @mock.patch("airflow.cli.commands.db_command.db.initdb")
     def test_cli_initdb(self, mock_initdb):
-        db_command.initdb(self.parser.parse_args(["db", "init"]))
-
+        with pytest.warns(
+            expected_warning=DeprecationWarning, match="ActionCommand `init` is deprecated"
+        ) as warning_record:
```

Comment:
```suggestion
        with pytest.warns(DeprecationWarning, match="`init` is deprecated") as warning_record:
```

Normally the expected warning is a positional arg. That coupled with the warning message changes above might allow this to land on 1 line?

Reply:
Indeed it does!